pkg/manifests/manifests_test.go

```diff
@@ -1194,6 +1194,74 @@ ingress:
 	}
 }
 
+func TestPrometheusQueryLogFileConfig(t *testing.T) {
+	testCases := []struct {
+		name             string
+		queryLogFilePath string
+		expected         string
+		errString        string
```

Comment:
It's less brittle to check the error presence rather than matching the error message.

```suggestion
		errExpected        bool
```

Reply:
Very true, implemented suggestion